tests/cmake/install/install_test_framework.pl

```diff
@@ -0,0 +1,49 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
```

Comment:
Is this generally useful (as in, outside of `tests/cmake`)?

Reply:
I'm not entirely sure how useful it is. I made this out of paranoia because I wanted to remove as many references to the OpenDDS source tree as possible. Some one might find it useful though, so I can move it to `tools/scripts/`.